helmfile/charts/prometheus-alerts/templates/rules/backup-status.yaml

```diff
@@ -0,0 +1,128 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.backupStatus}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "backup-status" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: backup-status
+    rules:
+    - alert: HarborBackup
+      annotations:
+        description: The job daily backup job harbor-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-harborbackup
+        summary: The daily backup job harbor-backup have failed over 24 hours.
+      expr: |-
+        (
+          (min((time()-kube_job_status_completion_time{job_name=~"harbor-backup-cronjob-.*", cluster=~".*"})/3600)) > 24
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: HarborBackup
+      annotations:
+        description: The job daily backup job harbor-backup have failed over 48 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-harborbackup
+        summary: The daily backup job harbor-backup have failed over 48 hours.
+      expr: |-
+        (
+          (min((time()-kube_job_status_completion_time{job_name=~"harbor-backup-cronjob-.*", cluster=~".*"})/3600)) > 48
+        )
+      for: 1h
+      labels:
+        severity: critical
```

Comment:
This goes for all the backup alerts.

I don't think we want any of these to be critical alerts. That would mean that we would be woken up during the night.
Also I would like some difference in the alert name to know which alert was triggered, feel free to chose something other that what I suggested here.
```suggestion
    - alert: HarborBackupFailedOnce
      annotations:
        description: The job daily backup job harbor-backup have failed over 24 hours.
        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-harborbackup
        summary: The daily backup job harbor-backup have failed over 24 hours.
      expr: |-
        (
          (min((time()-kube_job_status_completion_time{job_name=~"harbor-backup-cronjob-.*", cluster=~".*"})/3600)) > 24
        )
      for: 1h
      labels:
        severity: medium
    - alert: HarborBackupFailedTwice
      annotations:
        description: The job daily backup job harbor-backup have failed over 48 hours.
        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-harborbackup
        summary: The daily backup job harbor-backup have failed over 48 hours.
      expr: |-
        (
          (min((time()-kube_job_status_completion_time{job_name=~"harbor-backup-cronjob-.*", cluster=~".*"})/3600)) > 48
        )
      for: 1h
      labels:
        severity: warning
```